eris/tests/unit/sample/stest1.py

```diff
@@ -0,0 +1 @@
+#Sample test file
```

Comment:
I understand this it the initial step, but pass the pep8 tests,
Need space after # in block comment,
and at least need one valid code line.
you should change this to,
--- a/eris/tests/unit/sample/stest1.py
+++ b/eris/tests/unit/sample/stest1.py
@@ -1 +1,2 @@
-#Sample test file
+# Sample test file
+pass

Reply:
IMHO, we should use py35 jobs instead of py34.
Should update envlist  in tox.init,
--- a/tox.ini
+++ b/tox.ini
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.0
-envlist = py34,py27,pypy,pep8
+envlist = py35,py27,pypy,pep8

Reply:
one other thing, if you does not have special interest in pypy, then we should remove it from test envlist in tox.ini, as OpenStack does not support it anymore.
https://review.openstack.org/#/c/504748/

Reply:
Thanks for the comments!!! Deji & me will definitely look to fix.

Reply:
I made these changes and also removed pypy. Please review at your convenience